src/answers/answers.router.ts

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import { wrapAsync } from '../utils/wrapper';
+import { ValidateRequest } from '../utils/joi';
+import { AnswerController } from './answers.controller';
+import {
+  postSurveySchema, getSurveySchema,
+} from './validator/answers.schema';
+
+const AnswerRouter: Router = Router();
+
+AnswerRouter.post('/', ValidateRequest(postSurveySchema), wrapAsync(AnswerController.create));
+AnswerRouter.get('/find', ValidateRequest(getSurveySchema), wrapAsync(AnswerController.showSurveyById));
```

Comment:
change name of route
```suggestion
AnswerRouter.get('/getSurveysById', ValidateRequest(getSurveySchema), wrapAsync(AnswerController.showSurveyById));
```